boltzrpc/boltzrpc.proto

```diff
@@ -589,6 +589,7 @@ message CreateReverseSwapResponse {
 
 message CreateChainSwapRequest {
     // Amount of satoshis to swap. It is the amount expected to be sent to the lockup address.
+    // If left empty, any amount within the limits will be accepted.
```

Comment:
How about making it optional? It's a breaking change but not that bad and it's much nicer in other languages when you can omit the property instead of having to set 0